internal/server/admin/ca_certificate_test.go

```diff
@@ -182,6 +182,81 @@ func TestCACertificateCreate(t *testing.T) {
 	defer cleanup()
 	c := httpexpect.Default(t, s.URL)
 
+	t.Run("creating a certificate with subject fails", func(t *testing.T) {
+		cert := &v1.CACertificate{
+			Cert:    goodCACertOne,
+			Subject: "XXX",
+		}
+		res := c.POST("/v1/ca-certificates").WithJSON(cert).Expect()
+		res.Status(http.StatusBadRequest)
+		body := res.JSON().Object()
+		body.ValueEqual("message", "validation error")
+		body.Value("details").Array().Length().Equal(1)
+		err := body.Value("details").Array().Element(0).Object()
+		err.ValueEqual("type", v1.ErrorType_ERROR_TYPE_ENTITY.String())
+		err.Value("messages").Array().Element(0).String().
```

Comment:
nit: Should be checking the length of `$.details.messages` too (this comment is valid for most of these tests).